app/controllers/admin/settings/billing.js

```diff
@@ -0,0 +1,19 @@
+import Controller from '@ember/controller';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
+import { action } from '@ember/object';
+
+export default class extends Controller {
+  timezones = timezones;
+  @action
+  async updateInvoiceModel() {
+    this.set('isLoading', true);
+    let adminInvoiceModel = this.model;
```

Comment:
why store to a variable ?

Reply:
@niranjan94 Had overlooked this. Update it now.